daemon/src/maker_rollover.rs

```diff
@@ -0,0 +1,245 @@
+use crate::maker_cfd::RollOverCompleted;
+use crate::maker_inc_connections::TakerMessage;
+use crate::model::cfd::{OrderId, Role};
+use crate::model::Identity;
+use crate::oracle::GetAnnouncement;
+use crate::setup_contract::RolloverParams;
+use crate::tokio_ext::spawn_fallible;
+use crate::wire::{MakerToTaker, RollOverMsg};
+use crate::{
+    log_error, maker_inc_connections, oracle, schnorrsig, setup_contract, wire, Cfd, Stopping,
+};
+use anyhow::{Context as _, Result};
+use async_trait::async_trait;
+use futures::channel::mpsc;
+use futures::channel::mpsc::UnboundedSender;
+use futures::{future, SinkExt};
+use xtra::prelude::MessageChannel;
+use xtra::{Context, Handler, KeepRunning};
+use xtra_productivity::xtra_productivity;
+
+pub struct AcceptRollOver {
+    pub order_id: OrderId,
```

Comment:
An instance of this actor is for one specific `Cfd`, no need to send the order ID here. That is just an unnecessary error case. What if they don't match?